UncertainSCI/distributions.py

```diff
@@ -281,7 +281,7 @@ def __init__(self, flag=True, lbd=None, loc=None, mean=None,
         self.transform_standard_dist_to_poly = AffineTransform(A=A, b=b)
 
         if self.dim == 1:
-            A = np.array(lbd)
```

Comment:
Wasn't this `np` inclusion something that you included to fix a demo? Or am I confused and removing the `np` stuff is the fix?

Reply:
I've put in a new commit that added these changes back in after reverting them